src/postprocessors/Reactivity.C

```diff
@@ -0,0 +1,52 @@
+/********************************************************************/
+/*                  SOFTWARE COPYRIGHT NOTIFICATION                 */
+/*                             Cardinal                             */
+/*                                                                  */
+/*                  (c) 2021 UChicago Argonne, LLC                  */
+/*                        ALL RIGHTS RESERVED                       */
+/*                                                                  */
+/*                 Prepared by UChicago Argonne, LLC                */
+/*               Under Contract No. DE-AC02-06CH11357               */
+/*                With the U. S. Department of Energy               */
+/*                                                                  */
+/*             Prepared by Battelle Energy Alliance, LLC            */
+/*               Under Contract No. DE-AC07-05ID14517               */
+/*                With the U. S. Department of Energy               */
+/*                                                                  */
+/*                 See LICENSE for full restrictions                */
+/********************************************************************/
+
+#include "Reactivity.h"
+#include "openmc/eigenvalue.h"
+
+registerMooseObject("CardinalApp", Reactivity);
+
+InputParameters
+Reactivity::validParams()
+{
+  InputParameters params = OpenMCPostprocessor::validParams();
```

Comment:
Great start! I think you can reduce some code duplication if you have your `Reactivity` postprocessor inherit from `KEigenvalue` -- this would also allow you to just call `KEigenvalue::getValue()` to fetch k (and also allow users to choose which estimator for k they would like).